doc/releases/changelog-dev.md

```diff
@@ -192,6 +192,17 @@
 
 <h4>Work easily and efficiently with operators 🔧</h4>
 
+* We can compute the center of a dynamical Lie algebra.
+  [(#5477)](https://github.com/PennyLaneAI/pennylane/pull/5477)
+
+  Given a DLA `g`, we can now compute its center. The `center` is the collection of operators that commute with _all_ other operators in the DLA.
+
+  ```pycon
+  >>> g = [X(0), X(1) @ X(0), Y(1), Z(1) @ X(0)]
+  >>> qml.center(g)
+  [X(0)]
+  ```
+
```

Comment:
Not a blocker for this PR as we can do it separately, but this can fit into the DLA section.